base_layer/core/src/chain_storage/postgres/migrations/2020-04-07-15-40_inputs/up.sql

```diff
@@ -0,0 +1,13 @@
+create table inputs (
```

Comment:
Totally agree to drop this table and use the `created_in` field in the `tx_outputs` table